server/config/persist_options.go

```diff
@@ -449,6 +449,11 @@ func (o *PersistOptions) IsAuditEnabled() bool {
 	return o.GetPDServerConfig().EnableAudit
 }
 
+// IsRateLimitEnabled returns whether rate limit middleware is enabled
+func (o *PersistOptions) IsRateLimitEnabled() bool {
```

Comment:
Will this method be called in the future? It seems that there is no caller in this PR.